### PR TITLE
For jujuv3, actions are called functions

### PR DESCRIPTION
**Pull request by wallyworld**
## Description of change

Behind the juju-v3 feature flag, adjust the actions CLI to refer instead to functions.
This includes command names and output.

## QA steps

Run juju help 'cmdname'
Run the commands and checkout output

### CODE REVIEW

File: cmd/juju/action/call.go [hunk 1]
```diff
@@ -54,28 +54,27 @@ type callCommand struct {
 }
 
 const callDoc = `
-Run an Action for execution on a given unit, with a given set of params.
-The Action ID is returned for use with 'juju show-action-output <ID>' or
-'juju show-action-status <ID>'.
+Run a function for execution on a given unit, with a given set of params.
```
Comment (howbazaar) [inline cmd/juju/action/call.go]: Run a charm function?

File: cmd/juju/action/export_test.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func (c *CallCommand) UnitNames() []string {
 }
 
 func (c *CallCommand) ActionName() string {
```
Comment (howbazaar) [inline cmd/juju/action/export_test.go]: FunctionName?

File: cmd/juju/action/show.go [hunk 3]
```diff
@@ -71,7 +74,11 @@ func (c *showCommand) Info() *cmd.Info {
 		Doc:     showActionDoc,
 	})
 	if featureflag.Enabled(feature.JujuV3) {
+		info.Name = "show-function"
```
Comment (howbazaar) [inline cmd/juju/action/show.go]: Ick...

Comment (wallyworld) [reply cmd/juju/action/show.go]: Yeah, but it's the same command, just with a different name. Achieves the result with very little code. So smaller amount of churn when flag is removed.

File: cmd/juju/action/call.go (hunk 1 above)
Comment (wallyworld) [reply cmd/juju/action/call.go]: yup

File: cmd/juju/action/export_test.go (hunk 2 above)
Comment (wallyworld) [reply cmd/juju/action/export_test.go]: yup

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**